### PR TITLE
Fixes the variable names in the fire() procs for a few of the tickers

### PR DESCRIPTION
**Pull request by Iamgoofball**
this needed done, it'd get done eventually, figured I'd get it out of the way now

### CODE REVIEW

**Comment by Iamgoofball** [comment]
>doesn't compile

remove asteroid/ministation from travis 2015

**Comment by Carn** [comment]
people want ministation kept. There's also an issue for it, I've offered to help people update them. I think it was CorruptComputer doing asteroid and tkdrg doing ministation (I may be mistaken here)
Also, I did say I'd fix these myself. Stop trying to force everything.

**Comment by Iamgoofball** [comment]
I'm saving you time then as you don't need to fix it now as it's been fixed

**Comment by Carn** [comment]
You didn't fix anything.
You're just acting like a child

**Comment by optimumtact** [comment]
:+1: descriptive variables names is always nice

File: code/controllers/subsystem/pipenets.dm [hunk 1]
```diff
@@ -28,9 +28,9 @@ var/datum/subsystem/pipenets/SSpipe
 
 /datum/subsystem/pipenets/fire()
 	var/i=1
-	for(var/thing in networks)
-		if(thing)
-			thing:process()
+	for(var/pipe in networks)
```
Comment (Iamgoofball) [inline code/controllers/subsystem/pipenets.dm]: i'll swap it to pipe_network then

File: code/controllers/subsystem/shuttles.dm [hunk 2]
```diff
@@ -62,9 +62,9 @@ var/datum/subsystem/shuttle/SSshuttle
 	points += points_per_decisecond * wait
 
 	var/i=1
-	for(var/thing in mobile)
-		if(thing)
-			var/obj/docking_port/mobile/P = thing
+	for(var/docking_port in mobile)
```
Comment (Iamgoofball) [inline code/controllers/subsystem/shuttles.dm]: will swap for something else

**Comment by MrPerson** [comment]
There is absolutely no reason not to make the names more descriptive

**Comment by Carn** [comment]
That doesn't change the fact you're all being stupid.
Somebody could have just PMed me to talk about it like humans. Instead of it taking all maintainers to spam the same comment at me a zillion times even after I have agreed to change it.
It's just sad.

**Comment by Iamgoofball** [comment]
why do you insist on doing everything yourself

it's a group open source project

you don't have to solo everything, we can handle some of the small problems for you so you can work on more important problems

**Comment by phil235** [comment]
@Iamgoofball Can you squash your commits?

**Comment by Carn** [comment]
lol, you ended up closing it
10/10

**Comment by optimumtact** [comment]
:angry: :frog:

**Comment by Iamgoofball** [comment]
@Carn pr was out of date and needed a re-do

**Pull request closed without merging**